mlir/lib/Dialect/Vector/Transforms/VectorLinearize.cpp

```diff
@@ -82,7 +85,7 @@ struct LinearizeConstantLike final
 
   LinearizeConstantLike(
       const TypeConverter &typeConverter, MLIRContext *context,
-      unsigned targetVectBitWidth = std::numeric_limits<unsigned>::max(),
+      unsigned targetVectBitWidth = defaultTargetVectorBitWidth,
```

Comment:
Is this needed for the `vector.splat` pattern?

Reply:
no, this will go away once I rebase after #136581 & #138072